src/pages/monetization/app-review-process/index.mdx

```diff
@@ -61,7 +61,7 @@ If you want to use the word "LiveChat" in your app name, the only acceptable way
 
 Let's use an example from the <a href="https://developers.livechat.com/console/apps/explore" target="_blank" rel="noopener noreferrer">**App ideas**</a> section in Developer Console.
 
-<img alt="LiveChat App Ideas" src="/images/monetization/app-review-process/livechat-app-ideas.png" width="800px" height="485px"/>
+<img alt="LiveChat App Ideas" src="/images/monetization/app-review-process/App-review-process-app-ideas.png" width="1542px" height="913px"/>
```

Comment:
A few words about image naming:
- names shouldn't start with a capital latter
- it's best for SEO if image names contain the word "livechat"

Reply:
done:)